ietf-document/draft-0rtt-bdp/draft-kuhn-quic-0rtt-bdp-07.txt

```diff
@@ -5,20 +5,29 @@
 Internet Engineering Task Force                             N. Kuhn, Ed.
 Internet-Draft                                                      CNES
 Intended status: Informational                           E. Stephan, Ed.
-Expires: October 1, 2020                                          Orange
+Expires: November 9, 2020                                         Orange
                                                        G. Fairhurst, Ed.
                                                   University of Aberdeen
-                                                          March 30, 2020
+                                                             May 8, 2020
 
 
                Transport parameters for 0-RTT connections
                       draft-kuhn-quic-0rtt-bdp-07
 
 Abstract
 
-   0-RTT improves egress traffic when reconnecting.  This memo proposes
-   the sharing of additional transport parameters to improve ingress
-   traffic when reconnecting.
+   0-RTT mechanisms reduce the time it takes for the first bytes of
+   application data to be processed in a transport connection and can
+   greatly reduce connection latency during setup.  The 0-RTT transport
+   features described by quic-transport help clients establish secure
+   connections with a minimal number of round-trips.
+
+   This document describes a generic method to exchange path parameters
+   relating to transport.  The additional transport parameters can help
+   a connection that continues after an interruption by caching and
```

Comment:
Is connection continuing after interruption sufficiently broad (it could imply session semantics of the same session). I think we could be talking about a new connection that follows a recent connection to the same endpoint? 
That also impacts /resumes/ next, which could be "restarts"???? or something?

Reply:
I agree that this could be improved. I have added text in the commit "reword-abstract"